src/app/features/comments/comments-container/comment-container.component.ts

```diff
@@ -14,10 +15,14 @@ export class CommentContainerComponent implements OnInit {
   comments: Comment[];
   errorMessage = '';
 
-
   constructor(private commentService: CommentsService) {
   }
 
+
+  setNewCommentItem(newComment: NewComment): void {
```

Comment:
the term `set` is a bit misleading, cause it refers that the whole item is set, I would rather use the word `add` or `push` in method name instead of `set`